pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -213,7 +261,7 @@ func (r *ReconcileUserSignup) provisionMasterUserRecord(userSignup *toolchainv1a
 
 	mur := &toolchainv1alpha1.MasterUserRecord{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      userSignup.Spec.CompliantUsername,
+			Name:      userSignup.Status.CompliantUsername,
```

Comment:
Wouldn't `userSignup.Status.CompliantUsername` be always empty here? It's set only if MUR already exists: https://github.com/codeready-toolchain/host-operator/pull/93/files#diff-5b4b18a0fc2e0d708e4e6334cdb6bc69R157

Reply:
You're right, this is one spot that I missed. :)  Fixed now.